features/old/api/features.feature

```diff
@@ -8,59 +8,60 @@ Feature: Features :)
-    And I go to the edit page for plan "Basic"
-    And I press the disable button for feature "50% less bugs"
-    Then I should see disabled feature "50% less bugs"
-    And feature "50% less bugs" should be disabled for plan "Basic"
+    Then I go to the edit page for plan "Basic"
+    And I disable feature "50% more bugs"
+    Then feature "50% more bugs" should be disabled
 
-  @javascript @wip
+  @javascript
   Scenario: Enable a feature
-    Given a feature "Light-speed connection" of provider "foo.3scale.localhost"
-    And feature "Light-speed connection" is disabled for plan "Basic"
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    And a feature "50% less bugs" of provider "foo.3scale.localhost"
+
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
-    And I press the enable button for feature "Light-speed connection"
-    Then I should see enabled feature "Light-speed connection"
-    And feature "Light-speed connection" should be enabled for plan "Basic"
+    Then I go to the edit page for plan "Basic"
+    And I enable feature "50% less bugs"
+    Then feature "50% less bugs" should be enabled
 
-  @javascript @wip
+  @javascript
   Scenario: Edit a feature
-    Given a feature "Magic" of provider "foo.3scale.localhost"
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    And a feature "Magic" of provider "foo.3scale.localhost"
+
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
-    And I follow "Edit" for feature "Magic"
+    Then I go to the edit page for feature "Magic"
+    Then I should see "Edit Feature"
     And I fill in "Name" with "More magic"
-    And I press "Update Feature"
-    Then I should see feature "More magic"
-    And provider "foo.3scale.localhost" should have feature "More magic"
-    But provider "foo.3scale.localhost" should not have feature "Magic"
+    And I press "Save"
+    Then there is no feature named "Magic"
+    And there is feature named "More magic"
 
-  @javascript @wip
+  @javascript
   Scenario: Delete a feature
-    Given a feature "Invulnerability" of provider "foo.3scale.localhost"
-    When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
-    And I press "Delete" for feature "Invulnerability" and I confirm dialog box
-    Then I should not see feature "Invulnerability"
-    And provider "foo.3scale.localhost" should not have feature "Invulnerability"
+    Given the provider has "service_plans" switch allowed
+    Given a service plan "Basic" for service "API" exists
+    And a feature "Invulnerability" of provider "foo.3scale.localhost"
 
+    When I log in as provider "foo.3scale.localhost"
+    And I go to the service plans admin page
+    Then I go to the edit page for admin service plan "Basic"
+    And I follow "Delete" and I confirm dialog box
```

Comment:
The issue here is that it says `follow "Delete"` but the "Delete" is a button, not just a link, so it should be `I press "Delete"`